ui_framework/components/modal/confirm_modal.js

```diff
@@ -9,57 +9,98 @@ import { KuiModalBody } from './modal_body';
 import { KuiModalBodyText } from './modal_body_text';
 import { KuiButton } from '../index';
 
-export function KuiConfirmModal({
-    message,
-    title,
-    onCancel,
-    onConfirm,
-    cancelButtonText,
-    confirmButtonText,
-    className,
-    ...rest }) {
-  const ariaLabel = rest['aria-label'];
-  const dataTestSubj = rest['data-test-subj'];
-  return (
-    <KuiModal
-      style={{ 'width': '450px' }}
-      data-tests-subj={ dataTestSubj }
-      aria-label={ ariaLabel }
-      className={ className }
-    >
-      {
-        title ?
-          <KuiModalHeader>
-            <KuiModalHeaderTitle data-test-subj="confirmModalTitleText">
-              { title }
-            </KuiModalHeaderTitle>
-          </KuiModalHeader>
-          : null
-      }
-      <KuiModalBody>
-        <KuiModalBodyText data-test-subj="confirmModalBodyText">
-          { message }
-        </KuiModalBodyText>
-      </KuiModalBody>
+export const CONFIRM_BUTTON = 'confirm';
+export const CANCEL_BUTTON = 'cancel';
 
-      <KuiModalFooter>
-        <KuiButton
-          type="hollow"
-          data-test-subj="confirmModalCancelButton"
-          onClick={ onCancel }
-        >
-          {cancelButtonText}
-        </KuiButton>
-        <KuiButton
-          type="primary"
-          data-test-subj="confirmModalConfirmButton"
-          onClick={ onConfirm }
-        >
-          {confirmButtonText}
-        </KuiButton>
-      </KuiModalFooter>
-    </KuiModal>
-  );
+export const CONFIRM_MODAL_BUTTONS = [
```

Comment:
From what I can tell this `const` is only used within this module, as part of setting `KuiConfirmModal.propTypes`. Does it need to be exported?